Jenkinsfile-build

```diff
@@ -30,18 +29,18 @@ pipeline {
                 script {
                   tdr.reportStartOfBuildToGitHub(repo, env.GIT_COMMIT)
                 }
-                checkout([$class: 'GitSCM', branches: [[name: '*/master']], userRemoteConfigs: [[url: 'https://github.com/UKHomeOffice/keycloak-theme-govuk.git']]])
                 sh 'npm install'
-                sh 'npm run build'
-                stash includes: "govuk/**", name: "govuk"
+                sh "npx stylelint themes/tdr/css-src/sass/*.scss"
```

Comment:
I wonder if `themes/tdr/css-src/sass/**/*.scss` would be better? I think that means it scans any subdirectories, if there ever are some. I don't think we'd remember to update this line if we added more structure to the sass.

Reply:
I don't think `themes/tdr/css-src/sass/**/*.scss` does actually recursively check all the files. If you run it locally you only get warnings from the overrides directory listed under the `_all.scss` file

Reply:
After a bit of testing the following format recursively runs stylelint, including sub-directories: `themes/tdr/css-src/**/*.scss`

I deliberately dirtified the sass, and added a temporary additional sub-directory layer with a dirtified scss file in it. All the errors were picked up.

Reply:
Cheers. I didn't realise that `/**/` didn't include the base directory.